webapp/src/routes/[[lang]]/my/organizations/[id]/credential-issuances/[issuance_id]/+page.svelte

```diff
@@ -32,10 +32,10 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 	$: issuanceFlowQr = generateQr(
 		createIntentUrl({
 			credential_configuration_ids: [service.type_name],
-			credential_issuer: credential_issuer.endpoint,
+			credential_issuer: credential_issuer.endpoint.endsWith('/credential_issuer') ? credential_issuer.endpoint : `${credential_issuer.endpoint}/credential_issuer`,
```

Comment:
I see the intent to overcome `automagically` error prone endpoints, that is lovely. But you suppose that the credential_issuer is not correct if left at the end of a string. I would prefer to not suppose but maybe have a real check, like check that gives you a 404 or a 200... you can go with a small helper that does a simple HTTP method and checks the status. Then you choose the one that is correct for the better. Actually also accurate checks can be done, but guess that for now is super fine. Something like

```js
   credential_issuer: `${sanitizeCredentialIssuerEndpoint(credential_issuer.endpoint)}`
```

Reply:
Our credential issuers will always end with `/credential_issuer` it is something we always add when downloading the microservices